features/base/fontIcons/JitsiFontIcons.js

```diff
@@ -0,0 +1,9 @@
+import { createIconSetFromIcoMoon } from 'react-native-vector-icons';
```

Comment:
The following is merely a question, not a call for action.

Is the feature base/fontIcons intended to be platform-independent in the future? I'm asking because the import of react-native-vector-icons effectively makes this file native-specific but then we put these either with the file extension .native.js or in a native directory. So I'm currently wondering whether JitsiFontIcons.js and/or the fonts directory are platform-independent or native-specific & whether we should reflect that in their location like we do in other features.